api/server.go

```diff
@@ -77,8 +83,80 @@ func (s *Server) RatsdChares(w http.ResponseWriter, r *http.Request, param Ratsd
 		return
 	}
 
+	nonce, err := base64.RawURLEncoding.DecodeString(requestData.Nonce)
+	if err != nil {
+		errMsg := fmt.Sprintf("fail to decode nonce from the request: %s", err.Error())
+		p := &problems.DefaultProblem{
+			Type:   string(TagGithubCom2024VeraisonratsdErrorInvalidrequest),
+			Title:  string(InvalidRequest),
+			Detail: errMsg,
+			Status: http.StatusBadRequest,
+		}
+		s.reportProblem(w, p)
+		return
+	}
 	s.logger.Info("request nonce: ", requestData.Nonce)
+	s.logger.Info("request media type: ", *(param.Accept))
+
+	// Use a map until we finalize ratsd output format
+	eat := make(map[string]interface{})
+	collection := cmw.NewCollection("tag:ietf.org,2025:ratsd")
+	eat["eat_profile"] = TagGithubCom2024Veraisonratsd
+	eat["eat_nonce"] = requestData.Nonce
+	pl := s.manager.GetPluginList()
+	if len(pl) == 0 {
+		errMsg := "no sub-attester available"
+		p := problems.NewDetailedProblem(http.StatusInternalServerError, errMsg)
+		s.reportProblem(w, p)
+		return
+	}
+
+	for _, pn := range s.manager.GetPluginList() {
+		attester, err := s.manager.LookupByName(pn)
+		if err != nil {
+			errMsg := fmt.Sprintf(
+				"failed to get handle from %s: %s", pn, err.Error())
+			p := problems.NewDetailedProblem(http.StatusInternalServerError, errMsg)
+			s.reportProblem(w, p)
+			return
+		}
+
+		formatOut := attester.GetSupportedFormats()
+		if !formatOut.Status.Result || len(formatOut.Formats) == 0 {
+			errMsg := fmt.Sprintf("no supported formats from attester %s: %s ",
```

Comment:
I think we should fail hard rather than soft here: we should either return the complete view of the sub-attesters or an error.

Reply:
I've made it as a hard failure similar to other checks around this one.